src/components/Button.jsx

```diff
@@ -1,7 +1,25 @@
 import React from "react";
+import Link from "next/link";
 
 const Button = ({ text, link }) => {
-  return <div>Button</div>;
+  const baseStyle =
+    "rounded-full text-[2rem] font-plain text-center inline-block";
```

Comment:
- remove all [] they should all be existing functions. 
- no variables; move all the styling into into className={} 
- also it should be transparent with the landng (the blue button)